src/platform.rs

```diff
@@ -53,7 +53,7 @@ impl Platform {
             // TODO: consider using some kind of lookup table?
             arch: match arch.to_lowercase().as_str() {
                 "chipeightraw" | "chip8raw" | "c8r" | "chip8r" => PlatformArch::ChipEightRaw,
-                "chipeight" | "chip8" | "c8" | "chip8" => PlatformArch::ChipEight,
+                "chipeight" | "chip8" | "c8"  => PlatformArch::ChipEight,
```

Comment:
You accidentally added too (two) many spaces here, should just be one

Reply:
Oh, my bad! Will fix it right now.